tests/thin-client/steel-thread.py

```diff
@@ -2,6 +2,7 @@
 from snowflake.snowpark.functions import col, contains
 
 session = Session.builder.create()
-df = session.table('test_table')
-df = df.filter("STR LIKE '%e%'")
-df.show()
+df = session.table('test_table').filter("STR LIKE '%e%'")
+# col = df.col("STR") # TODO: determine whether this should create and assign statement or not
+df2 = df.select("A", df.col("STR"), col("B"), df.STR, col("A") + col("B"))
+print(session._ast_batch._request)
```

Comment:
Could you attach the output to the PR description please? I wanted to double-check the result.